liblocation/src/main/java/com/mapbox/android/core/location/LocationEngine.java

```diff
@@ -129,7 +129,8 @@ public int getInterval() {
   /**
    * Set the rate in milliseconds at which your app prefers to receive location updates. Note that the location updates
    * may be faster than this rate if another app is receiving updates at a faster rate, or slower than this rate, or
-   * there may be no updates at all (if the device has no connectivity, for example).
+   * there may be no updates at all (if the device has no connectivity, for example). This has no effect on
+   * AndroidLocationEngine, as it uses fastestInterval to set minTime.
```

Comment:
It would have been great to add `<tt></tt>` around `AndroidLocationEngine` i.e. `<tt>AndroidLocationEngine</tt>` to format/style it as code when docs are generated.

Reply:
being handled here -> https://github.com/mapbox/mapbox-events-android/pull/114